cmd/tier-journal.go

```diff
@@ -42,15 +42,20 @@ type tierJournal struct {
 }
 
 type jentry struct {
-	ObjName  string `msg:"obj"`
-	TierName string `msg:"tier"`
+	ObjName         string `msg:"obj"`
+	RemoteVersionID string `msg:"rvid"`
```

Comment:
```suggestion
	VersionID string `msg:"vid"`
```
`jentry` values refer only to remote objects.